src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -65,18 +58,10 @@ public String getLanguage() {
         return userChoice;
     }
 
-    public Language createLanguageOptions(String userChoice) {
-        if (userChoice.equals("1")) {
-            return new English();
-        } else {
-            return new Greek();
-        }
-    }
-
     public void setLanguage() {
         askForLanguage();
         String userChoice = getLanguage();
-        this.language = createLanguageOptions(userChoice);
+        this.language = LanguageSetter.setLanguage(userChoice);
```

Comment:
I feel like here the data structure `String` is misleading the content of the variable userChoice which is an id which could be an int. Maybe `languageIdSelected` could be more explicit or change the data structure used?